src/EspMQTTClient.cpp

```diff
@@ -131,6 +154,11 @@ void EspMQTTClient::enableLastWillMessage(const char* topic, const char* message
   _mqttLastWillRetain = retain;
 }
 
+void EspMQTTClient::enableLastWillMessage(const String &topic, const String &message, const bool retain )
```

Comment:
I think this method is unrelated to this PR? Also as every method currently expects `const char*` it should be that way for everything.

I would remove it